lib/forest_liana/bootstrapper.rb

```diff
@@ -38,13 +38,34 @@ def display_apimap
 
     private
 
+    def get_collection(collection_name)
+      ForestLiana.apimap.find { |collection| collection.name.to_s == collection_name }
+    end
+
+    def get_action(collection, action_name)
+      collection.actions.find {|action| action.name == action_name}
+    end
+
     def generate_apimap
       create_apimap
       require_lib_forest_liana
       format_and_validate_smart_actions
 
       if Rails.env.development?
         @collections_sent = ForestLiana.apimap.as_json
+
+        @collections_sent.each do |collection|
+          collection['actions'].each do |action|
+            unless action['hooks'].nil?
+              collection = get_collection(collection['name'])
+              a = get_action(collection, action['name'])
+              load = a.hooks.key?(:load) && a.hooks[:load].is_a?(Proc)
+              change = a.hooks.key?(:change) && a.hooks[:change].is_a?(Hash) ? a.hooks[:change].keys : []
+              action['hooks'] = {:load => load, :change => change}
+            end
+          end
+        end
+
```

Comment:
I'm not sure if it is the right place for doing this business code, but I did not find another way :/